bootstrap/api/v1beta2/ck8sconfig_types.go

```diff
@@ -90,6 +94,65 @@ type CK8sControlPlaneConfig struct {
 	// ExtraKubeAPIServerArgs is extra arguments to add to kube-apiserver.
 	// +optional
 	ExtraKubeAPIServerArgs map[string]*string `json:"extraKubeAPIServerArgs,omitempty"`
+
+	// Annotations is a map of annotations to add to the control plane node.
+	// +optional
+	Annotations map[string]string `json:"annotations,omitempty"`
```

Comment:
Let's move this to `InitConfiguration` as well. We do not intend on making this field mutable (as in: changing this on a deployed cluster will not have any effect).